src/MICore/CommandFactories/lldb.cs

```diff
@@ -177,6 +177,18 @@ public override Task Catch(string name, bool onlyOnce = false, ResultClass resul
             throw new NotImplementedException("lldb catch command");
         }
 
```

Comment:
Would leave a comment here to update these commands if the jump command is implemented in [`lldb-mi`](https://github.com/lldb-tools/lldb-mi/tree/master/src)